Firestore/core/test/unit/bundle/bundle_serializer_test.cc

```diff
@@ -26,10 +38,165 @@ namespace bundle {
 namespace {
 
 using google::protobuf::Message;
+using google::protobuf::util::MessageDifferencer;
 using google::protobuf::util::MessageToJsonString;
 using nlohmann::json;
+using ProtoBundledDocumentMetadata = ::firestore::BundledDocumentMetadata;
 using ProtoBundleMetadata = ::firestore::BundleMetadata;
-using util::ReadContext;
+using ProtoDocument = ::google::firestore::v1::Document;
+using ProtoMaybeDocument = ::firestore::client::MaybeDocument;
+using ProtoNamedQuery = ::firestore::NamedQuery;
+using ProtoValue = ::google::firestore::v1::Value;
+using core::Query;
+using local::LocalSerializer;
+using model::DatabaseId;
+using nanopb::ByteString;
+using nanopb::MakeByteString;
+using nanopb::ProtobufParse;
+using remote::Serializer;
+using std::numeric_limits;
+using testutil::Array;
+using testutil::Filter;
+using testutil::Map;
+using testutil::OrderBy;
+using testutil::Value;
+
+class BundleSerializerTest : public ::testing::Test {
+ public:
+  BundleSerializerTest()
+      : remote_serializer(DatabaseId("p", "default")),
+        local_serializer(remote_serializer),
+        bundle_serializer(remote_serializer) {
+    msg_diff_.ReportDifferencesToString(&message_differences);
+  }
+
+  remote::Serializer remote_serializer;
+  local::LocalSerializer local_serializer;
+  bundle::BundleSerializer bundle_serializer;
+
+  static std::string GetDocumentFullPath(const std::string& path) {
```

Comment:
In the interest of reducing clutter in tests, you could call this just `FullPath`.

Reply:
Done.